.github/workflows/voter.yml

```
@@ -17,7 +17,7 @@ jobs:
 
     strategy:
       matrix:
-        java: ['11.0']
+        java: ['17.0']
```

why not testing both? would be good to support 11, too. at least for a while.

```suggestion
        java: ['11.0', '17.0']
```